src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,63 @@
+import rightIcon from "../../assets/icon/arrow-right-icon.svg";
+import leftIcon from "../../assets/icon/arrow-left-icon.svg";
+import styles from "./Pagination.module.css";
+import { useEffect, useState } from "react";
+
+const Pagination = ({ currentPage, totalCount, pageSize, onPageChange }) => {
+  const totalPages = Math.ceil(totalCount / pageSize);
+  const groupSize = 5;
+  const currentGroup = Math.ceil(currentPage / groupSize);
+  const startPage = (currentGroup - 1) * groupSize + 1;
+  const endPage = Math.min(startPage + groupSize - 1, totalPages);
+
+  if (totalPages <= 1) return null;
+
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < totalPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <div className={styles.pagination}>
+      <button
+        disabled={currentPage === 1}
+        onClick={handlePrevPage}
+        className={styles.navButton}
+      >
+        <img src={leftIcon} alt="왼쪽 화살표" className={styles.leftIcon} />
+      </button>
+
+      {Array.from({ length: endPage - startPage + 1 }, (_, i) => {
+        const page = startPage + i;
+        return (
+          <button
```

Comment:
💊 제안
button type 명시해주시면 더 좋겠습니다~

```suggestion
          <button
            type="button"
```